src/main/controllers/features/limited/isThisYourCompanyController.ts

```diff
@@ -34,6 +34,9 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             detailsAnswers.businessName = company.companyName;
             detailsAnswers.companyNumber = company.companyNumber;
             detailsAnswers.businessAddress = company.registeredOfficeAddress?.addressLineOne! +
+            "<br>" + company.registeredOfficeAddress?.addressLineTwo! +
```

Comment:
A check is needed here to make sure the added lines are not undefined otherwise the answers page will display undefined